src/py21cmfast/wrapper.py

```diff
@@ -232,6 +232,9 @@ class UserParams(StructWithDefaults):
         3: PEEBLES
         4: WHITE
         5: CLASS output
+
+    L_FACTOR : float, optional
+        Factor relating cube length to filter radius = (4PI/3)^(-1/3), default is 0.620350491
```

Comment:
This docstring is confusing -- how could a constant like this be a parameter?

Reply:
@andreimesinger can you comment on this? I'm not sure if L_FACTOR should be a variable.

Reply:
@steven-murray so I need to use L_FACTOR to initialize R_BUBBLE_MIN, which is the reason I've moved L_FACTOR into the wrapper. As @andreimesinger said, it's a constant and shouldn't be put in parameters. Would you prefer to putting it as the number directly in the code (.c and .py) or simply remove the docstring (so still store it in UserParams but don't allow users to change it)?

Reply:
I think in this case, the easiest thing is just to set `L_FACTOR = (4*np.pi/3)**(1./3)` in the `R_BUBBLE_MIN` property, since it will never change. It's slight duplication (i.e. it's defined in C and Python), but it's very small and much simpler this way.

Reply:
please see if the current way makes sense. I would do pr_feature now.